### PR TITLE
feat: add respective licenses to each package

### PR DESCRIPTION
**Pull request by almndbtr**
## Summary

Closes https://github.com/ourzora/zorb/issues/17

This introduces a license per package based on my interpretation of [this issue comment](https://github.com/ourzora/zorb/issues/17#issuecomment-1004539211) from Iain, specifically:

- the contract-related packages (Solidity) include a [GPLv3 license](https://www.gnu.org/licenses/gpl-3.0.en.html)
- the web-related (JS) packages include the [MIT license](https://mit-license.org/)

This also updates the README with a helpful pointer for casual readers:

> For full license details, please refer to the license file in each package's directory.

### Reviewer Notes 👓

- I am not a lawyer. I am just a Zorb enthusiast who wants to see this project and the @ourzora team win.
- For license guidance, I consulted https://choosealicense.com/ and the GitHub Docs' [Licensing a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository) guide.
- For license metadata, I consulted the [npm Docs | package.json - license](https://docs.npmjs.com/cli/vm7/configuring-npm/package-json#license) and the [SPDX License List](https://spdx.org/licenses/).
- Last but not least: I don't expect this to be merged as-is. I'd love feedback from the team to make sure we're doing the right thing by @ourzora and the people! 🟣

### CODE REVIEW

**Comment by almndbtr** [comment]
👀 I see some merge conflicts. I can address those!

**Comment by iainnash** [comment]
This looks great! i'll merge in once the merge conflicts are resolved.

File: packages/zorb-contracts/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "author": "iain nash",
-  "license": "GPLv3",
+  "license": "GPL-3.0-only",
```
Comment (almndbtr) [inline packages/zorb-contracts/package.json:7]: This change is informed by the official SPDX guidelines for this license:

https://spdx.org/licenses/GPL-3.0-only.html

**Comment by almndbtr** [comment]
> This looks great! i'll merge in once the merge conflicts are resolved.

@iainnash - Thank you! 🙇‍♂️ This is ready for re-review. 👓

**Comment by almndbtr** [comment]
@iainnash - Thank you so much!

**Pull request merged**